README.md

```diff
@@ -1213,7 +1213,7 @@ cooper-mbp:black cooper$ ~/venvs/b/bin/black-primer
 
 Failed Projects:
 
-## flake8-bugbear:
+### flake8-bugbear:
```

Comment:
This is a dirty fix so the docs can build successfully. We could just change the output of `black-primer` to not write `##` before failed projects.